backend/alembic/versions/2023_01_05_1745-20cd871f4ec7_added_user_to_textlabels.py

```diff
@@ -0,0 +1,30 @@
+"""Added user to TextLabels
+
+Revision ID: 20cd871f4ec7
+Revises: 8d269bc4fdbd
+Create Date: 2023-01-05 17:45:15.696468
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "20cd871f4ec7"
+down_revision = "8d269bc4fdbd"
```

Comment:
we need to fix the alembic tree, if you touch this PR .. could you please change down_revision to the latest?

Reply:
This should be fixed now but feel free to double check it's correct as I'm not too familiar with alembic